openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -157,9 +157,9 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
 
             <p class="openassessment_description" id="openassessment_step_select_description">
                 {% if 'example_based_assessment' in editor_assessments_order %}
-                    {% trans "In this assignment, you can include steps for learner training, peer assessment, self assessment, and example based assessment. Select the steps that you want below, and then drag them into the order that you want. If you include an example based assessment step, it must precede all other steps. If you include a learner training step, it must precede peer and self assessment steps." %}
+                    {% trans "In this assignment, you can include learner training, peer assessment, self assessment, example based assessment, and staff assessment steps. Select the steps that you want below, and then drag them into the order you want. If you include an example based assessment step, it must precede all other steps. If you include a learner training step, it must precede peer and self assessment steps. If you include a staff assessment step, it must be the final step. The type and number of assessment steps cannot be changed after a problem has been released." %}
```

Comment:
@efischer19 @explorerleslie 
Minor edits to the last sentence to make it active rather than passive voice:
"After you publish an ORA assignment, you cannot change the type and number of assessment steps."